remail/email_api/test_email_api.py

```
@@ -0,0 +1,52 @@
+        body="Test!!",
+        recipients=[EmailReception(contact=(Contact(email_address ="praxisprojekt-remail@uni-due.de")), kind=RecipientKind.to)],
+        #attachments=[Attachment(filename=r"C:\Users\toadb\Documents\ReinventingEmail\test.txt")]
+    )
+    exchange_test_email = Email(
+        
+        subject="test_exchange_mail",
+        body="Test!!",
+        recipients=[EmailReception(contact=(Contact(email_address ="thatchmilo35@gmail.com")), kind=RecipientKind.to)],
+        #attachments=[Attachment(filename=r"C:\Users\toadb\Documents\ReinventingEmail\test.txt")]
+    )
+    date = datetime.now(get_localzone())
```

Ruff (0.8.0)

`datetime` may be undefined, or defined from star imports

(F405)